### DISCUSSION TITLE
No support for Japanese

### DISCUSSION THREAD

**Discussion by zhuman90** [malzariey/filament-daterangepicker-filter#102] [Q&A]
No support for Japanese
![20240704113833](https://github.com/malzariey/filament-daterangepicker-filter/assets/12763586/fa25c3ff-a84a-4825-a6df-9a94b7de39c3)

**Comment by malzariey** [comment]
You are welcome to submit a PR to add a japanese translation file 

for example this PR https://github.com/malzariey/filament-daterangepicker-filter/pull/99
